tests/chrome-extension/background-script/service-worker.js

```diff
@@ -0,0 +1,32 @@
+// This is the service worker script, which executes in its own context
+// when the extension is installed or refreshed (or when you access its console).
+// It would correspond to the background script in chrome extensions v2.
+
+console.log("This prints to the console of the service worker (background script)")
+
+// Importing and using functionality from external files is also possible.
+import { Analytics } from "./rudderAnalytics.js";
+
+console.log(Analytics);
+
+const rudderClient = new Analytics(
+    "2EzJEim6arwPqWRy0dEp3ts1Czk",
+    "https://hosted-dev-dataplane.dev-rudder.rudderlabs.com/v1/batch",
+    {
+        logLevel: 'silly'
+    }
+);
```

Comment:
Let's put dummy values for the write key and data plane URL.

Reply:
fixed